core/domain/prod_validation_jobs_one_off_test.py

```diff
@@ -244,11 +244,11 @@ def setUp(self):
         self.item.put()
 
     def test_error_is_raised_if_fetch_external_properties_is_undefined(self):
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaisesRegexp(NotImplementedError, ''):
```

Comment:
I'm sorry but this will not do. There is no point in introducing this lint check if devs are going to bypass it anyway with an empty string.

Please make sure all strings are nonempty and have a reasonable chunk of the error message. Please also add a self.assertRaisesRegexp() method in the base test class which checks that the second arg is nonempty, and that warns the developers and raises a breaking exception if it is, and then calls its superclass method.

EDIT: it looks like @DubeySandeep [commented](https://github.com/oppia/oppia/pull/9860#discussion_r452707170) on this already, but the only response I can see (to his first comment about that) was "Ditto", so I still don't understand the root issue. If the problem is that there's no actual exception message defined, then please update the code to provide a proper exception message that makes sense in context.